### PR TITLE
eslint@^6.0.0 support

### PR DESCRIPTION
**Pull request by megheaiulian**
Supports eslint@^6.0.0
Adds chunking support to allow posting more than 50 annotations

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -124,39 +126,68 @@ const headers = {
...
+			await updateChecks(id, await eslint());
 		} catch (err) {
-			await updateCheck(id, 'failure');
+			await updateCheck(id, {
+				conclusion: 'failure',
+				status: 'completed',
+				completed_at: (new Date()).toISOString(),
+				output: {
+					title: checkName,
+					summary: 'Something went wrong while performing the check ...'
```
Comment (cristinecula) [inline index.js]: Maybe include the err.message in the summary?

File: index.js [hunk 2]
```diff
@@ -124,39 +126,68 @@ const headers = {
...
+					status: 'completed',
+					completed_at: (new Date()).toISOString(),
+					conclusion: errorCount > 0 ? 'failure' : 'success',
+					output: {
+						title: checkName,
+						summary: `${ errorCount } error(s), ${ warningCount } warning(s) found`,
+						annotations
+					}
+				}
+			);
```
Comment (cristinecula) [inline index.js]: I would put the ternary on it's own line.
```suggestion
			return updateCheck(
				id, 
				i < chunksLength - 1
					? {
						status: 'in_progress',
						output: {
							title: checkName,
							summary: `${ errorCount } error(s), ${ warningCount } warning(s) found`,
							annotations
						}
					}
					: {
						status: 'completed',
						completed_at: (new Date()).toISOString(),
						conclusion: errorCount > 0 ? 'failure' : 'success',
						output: {
							title: checkName,
							summary: `${ errorCount } error(s), ${ warningCount } warning(s) found`,
							annotations
						}
					}
			);
```

Comment (megheaiulian) [reply index.js]: Updated this because I needed to handle the case when there are no annotations.

File: index.js (hunk 1 above)
Comment (megheaiulian) [reply index.js]: Yes.

File: index.js [hunk 3]
```diff
@@ -163,7 +163,7 @@ const headers = {
 		await updateCheck(id, {
 			status: 'completed',
 			completed_at: (new Date()).toISOString(),
-			conclusion: errorCount > 0 ? 'failure' : 'success',
+			conclusion: errorCount > 0 ? 'failure' : warningCount > 0 && 'neutral' || 'success',
```
Comment (cristinecula) [inline index.js]: This is great! : +1:

Might read better like this:
```suggestion
			conclusion: errorCount > 0 
				? 'failure' 
				: warningCount > 0 
					? 'neutral' 
					: 'success',
```

Comment (megheaiulian) [reply index.js]: eslint rules disallow nested ternary...

Comment (cristinecula) [reply index.js]: :cry:

**Pull request merged**